keras_cv/metrics/coco/utils.py

```diff
@@ -32,7 +32,7 @@ def bounding_box_area(boxes):
     """
     w = boxes[..., bounding_box.RIGHT] - boxes[..., bounding_box.LEFT]
     h = boxes[..., bounding_box.BOTTOM] - boxes[..., bounding_box.TOP]
-    return tf.math.multiply(w, h)
+    return tf.math.abs(tf.math.multiply(w, h))
```

Comment:
Humm, any reason the math.abs is needed? Is it trying to handle any invalid data (eg right < left)?

Reply:
Sorry- should've reverted that.  That was a sanity check